package.json

```diff
@@ -38,7 +42,56 @@
     "grunt-webpack": "^3.0.2",
     "ink-docstrap": "^1.3.0",
     "matchdep": "~2.0.0",
+    "source-map-loader": "^0.2.4",
+    "uglify-js": "^3.4.9",
```

Comment:
This may cause issues, a conflict issue with `grunt-contrib-uglify` was reverted a couple of days ago via https://core.trac.wordpress.org/changeset/43686/

Reply:
It doesn't say why it was reverted. But it might not be a problem because `node_modules` can deal with different versions of the same package. At least for dev dependencies.